pkg/logproto/logproto.proto

```diff
@@ -55,6 +55,11 @@ message StreamRatesResponse {
   repeated StreamRate streamRates = 1;
 }
 
+message StreamMetadata {
+  uint64 streamHash = 1;
```

Comment:
I think you should include all you think you'll need from the start. But happy to do it later.

Not sure the lastSeenAt is required since the record is already stamp with produced time.

Reply:
Yeah, we talked about this yesterday. I think we can use the timestamp of the record and avoid embedding our own timestamps.